.devcontainer/recipes/postCreateCommand.sh

```
@@ -0,0 +1,3 @@
+uv tool install pre-commit --with pre-commit-uv --force-reinstall
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)